<?xml version="1.0" encoding="utf-8"?>
<ui version="4.0">
 <author>
    SPDX-FileCopyrightText: none
    SPDX-License-Identifier: GPL-3.0-or-later
  </author>
 <class>KisEditProfilesDialog</class>
 <widget class="QWidget" name="KisEditProfilesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>667</width>
    <height>682</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item colspan="4" column="0" row="2">
    <widget class="QListView" name="profileList"/>
   </item>
   <item column="0" row="1">
    <widget class="QPushButton" name="duplicateButton">
     <property name="text">
      <string>Duplicate</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item column="3" row="1">
    <widget class="QPushButton" name="resetButton">
     <property name="text">
      <string>Reset All</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item column="1" row="1">
    <widget class="QPushButton" name="removeButton">
     <property name="text">
      <string>Remove</string>
     </property>
    </widget>
   </item>
   <item column="2" row="1">
    <widget class="QPushButton" name="renameButton">
     <property name="text">
      <string>Rename</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>